instantout/actions.go

```diff
@@ -59,6 +59,7 @@ type InitInstantOutCtx struct {
 	initationHeight int32
 	outgoingChanSet loopdb.ChannelSet
 	protocolVersion ProtocolVersion
+	sweepAddress    string
```

Comment:
nit: alternatively could use `*btcutil.Address` pointer so deconding can happen on the CLI.